models/rating.js

```diff
@@ -14,8 +14,8 @@ const ratings = (sequelize, DataTypes) => {
         type: DataTypes.INTEGER,
         allowNull: false
       },
-      articleId: {
-        type: DataTypes.INTEGER,
+      articleSlug: {
```

Comment:
@ericnyirimana considering the name of this attribute ```articleSlug``` you want it to contain the slug of the article being rated. but with the association made I believe it will have the id of the article.

Reply:
@Kabalisa I have resolved it and I have added the targetted key for slug